### DISCUSSION TITLE
Is there an option for custom client to start automatically on reboot?

### DISCUSSION THREAD

**Discussion by pvasilek** [rustdesk/rustdesk-server-pro#448] [Q&A]
Hello,

We have a fleet of devices which need to be managed remotely without user interaction.
So, I am looking but can not find an option for custom client to start application and service automatically on reboot with screen sharing enabled.

**Comment by 21pages** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#android-start-on-boot-and-limitations, seems impossible for custom clients.

**Comment by pvasilek** [comment]
So, even if permissions may be configured via adb it does not have OnBoot handler to start? Is it something can be added soon?
